### PR TITLE
✨(slack) add our best practices when using Slack

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

We had a small discussion about the way we use of Slack that inspired me.

## Proposal

Advocating asynchronicity and using threads.

### CODE REVIEW

**Review by madmatah — APPROVED** [review]

**Review by jmaupetit — APPROVED** [review]

File: slack.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# FUN with Slack
+
+In this section, we will describe our best practices when using Slack.
+
+## Asynchronicity
+
+Slack is used:
+- for real-time discussion when two or more persons are actively connected at the same time,
+- to replace emails when sharing information or for asynchronous conversions.
```
Comment (lunika) [inline slack.md:9]:
```suggestion
- to replace emails when sharing information or for asynchronous conversations.
```
you mean conversation no ?

Comment (sampaccoud) [reply slack.md:9]: oups... yes!

**Review by lunika — APPROVED** [review]

**Review by jbpenrath — APPROVED** [review]

**Review by carofun — APPROVED** [review]

**Pull request merged**